src/OrchardCore.Modules/OrchardCore.AuditTrail/Services/IAuditTrailManager.cs

```diff
@@ -0,0 +1,67 @@
+using OrchardCore.AuditTrail.Models;
+using OrchardCore.AuditTrail.Providers;
+using OrchardCore.AuditTrail.Services.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrchardCore.AuditTrail.Services
+{
+    /// <summary>
+    /// Central service for the Audit Trail feature.
+    /// </summary>
+    public interface IAuditTrailManager
+    {
+        /// <summary>
+        /// Records an audit trail event.
+        /// </summary>
+        /// <typeparam name="T">The audit trail event provider type to determine the scope of the event name.</typeparam>
```

Comment:
Doc typeparam name does not match the method generic type name. Should be TAuditTrailEventProvider.

Tiny comment, but causing an error in my visual studio that is stopping this build.